### PR TITLE
migration(plugins): removes all plugins except slack and PD

### PR DESCRIPTION
**Pull request by scefali**
Paired with https://github.com/getsentry/sentry/pull/15468

Must be merged after: https://github.com/getsentry/getsentry/pull/3309

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -11,10 +11,6 @@ def pytest_configure(config):
     # Note: We could manually register/configure INSTALLED_APPS by traversing our entry points
     # or package directories, but this is easier assuming Sentry doesn't change APIs.
     # Note: Order of operations matters here.
-    from sentry.runner.importer import install_plugin_apps
-
-    install_plugin_apps("sentry.apps", settings)
```
Comment (joshuarli) [inline conftest.py]: Why did this get removed? Do slack/pd not need to be added to INSTALLED_APPS by sentry's importer?

**Review by wedamija — APPROVED** [review]
Can you post in #engineering and let people know they'll need to update their sentry-plugins if they pull master and are running getsentry? Just want to make sure we don't break people's devs again

File: conftest.py (hunk 1 above)
Comment (wedamija) [reply conftest.py]: Actually yeah, seems like we should keep this since there are still a couple plugins in `sentry.apps`

**Review by joshuarli — CHANGES_REQUESTED** [review]
https://github.com/getsentry/sentry-plugins/pull/538#discussion_r343282061

File: conftest.py (hunk 1 above)
Comment (scefali) [reply conftest.py]: @wedamija @joshuarli Hmmm? The only ones in that list were Jira, Jira AC, and SessionStack. All of those are now in sentry-core which has the entrypoint of `sentry.new_apps`.

Comment (joshuarli) [reply conftest.py]: @scefali Ah you're right, I understand now.

**Review by joshuarli — APPROVED** [review]

**Pull request merged**